android/app-example/src/main/java/com/badoo/ribs/example/rib/blocker/builder/BlockerModule.kt

```diff
@@ -0,0 +1,51 @@
+package com.badoo.ribs.example.rib.blocker.builder
+
+import com.badoo.ribs.core.Node
+import com.badoo.ribs.core.view.ViewFactory
+import com.badoo.ribs.example.rib.blocker.Blocker
+import com.badoo.ribs.example.rib.blocker.Blocker.Output
+import com.badoo.ribs.example.rib.blocker.BlockerInteractor
+import com.badoo.ribs.example.rib.blocker.BlockerRouter
+import com.badoo.ribs.example.rib.blocker.BlockerView
+import dagger.Provides
+import io.reactivex.functions.Consumer
+
+@dagger.Module
+internal object BlockerModule {
+
+    @BlockerScope
+    @Provides
+    @JvmStatic
+    internal fun router(
+        // pass component to child rib builders, or remove if there are none
+        component: BlockerComponent
+    ): BlockerRouter =
+        BlockerRouter()
```

Comment:
General question, how about creating a default empty router?

Reply:
Yes, I thought about it too.